### PR TITLE
Split Bambu Lab PLA filaments into categories to be able to use the finish property

### PR DESCRIPTION
**Pull request by dresende**
Will add gradient later.

### CODE REVIEW

**Comment by Donkie** [comment]
Thank you! I pushed just now an update to the schema that adds a few more fields which seem relevant here. I added the "pattern", "translucent" and "glow" fields. See the README for details

**Comment by dresende** [comment]
Ok, will take a look.

File: filaments/bambulab.json [hunk 1]
```diff
@@ -160,7 +179,26 @@
                 {
                     "name": "Charcoal",
                     "hex": "000000"
-                },
+                }
+            ]
+        },
+        {
+            "name": "{color_name}",
```
Comment (Donkie) [inline filaments/bambulab.json:186]: Here you can remove the Silk prefix from all filaments and put it here like `Silk {color_name}`

File: filaments/bambulab.json [hunk 2]
```diff
@@ -192,7 +230,26 @@
                 {
                     "name": "Silk Copper",
                     "hex": "5E4B3C"
-                },
+                }
+            ]
+        },
+        {
+            "name": "{color_name}",
```
Comment (Donkie) [inline filaments/bambulab.json:237]: Same here, remove Metallic suffix and add it here `{color_name} Metallic`

**Comment by Donkie** [comment]
I don't mind either way. You missed to remove the Translucent prefix though :)

**Comment by dresende** [comment]
> I don't mind either way. You missed to remove the Translucent prefix though :)

It seems Bambu changed PETG Translucent colors and added a "Clear" one.

**Comment by dresende** [comment]
Or I missed it before..

**Comment by dresende** [comment]
Had these filaments here so I might as well just add them.

**Comment by dresende** [comment]
Looks like the workflow does not accept gradients yet, or I made some kind of mistake..

**Comment by Donkie** [comment]
You only have 5 F's on line 435, should be 6 :)

**Comment by Donkie** [comment]
Thanks!

**Pull request merged**